examples/step-67/doc/results.dox

```diff
@@ -0,0 +1,390 @@
+<h1>Results</h1>
+
+<h3>Program output</h3>
+
+Running the program with the default settings on a machine with 40 processes
+produces the following output:
+@code
+Running with 40 MPI processes
+Vectorization over 8 doubles = 512 bits (AVX512), VECTORIZATION_LEVEL=3
+Number of degrees of freedom: 589824 ( = 4 [vars] x 4096 [cells] x 36 [dofs/cell/var] )
```

Comment:
If you use the approach here https://github.com/dealii/dealii/blob/master/examples/step-32/step-32.cc#L1873-L1879, then you get a comma separator:
```suggestion
Number of degrees of freedom: 589,824 ( = 4 [vars] x 4096 [cells] x 36 [dofs/cell/var] )
```